src/doc/man/includes/options-target-dir.md

```diff
@@ -1,6 +1,9 @@
 {{#option "`--target-dir` _directory_"}}
 Directory for all generated artifacts and intermediate files. May also be
 specified with the `CARGO_TARGET_DIR` environment variable, or the
-`build.target-dir` [config value](../reference/config.html). Defaults
-to `target` in the root of the workspace.
+`build.target-dir` [config value](../reference/config.html).
+{{#if temp-target-dir}} Defaults to a new temporary folder located in the
+temporary directory of the platform.
```

Comment:
There is some subtly here that I think should be included. When using `cargo install --path`, it will use the target directory in the workspace. Can you include that, too?

Reply:
Thanks for spotting this 😄. The text has been updated.